### ISSUE TITLE
Security records photos of crewmembers is broken

### ISSUE THREAD

**Issue by Fronsis** [BeeStation/BeeStation-Hornet#8328]
Title. Basically you can't see the pictures that are attached to the crewmembers on their security record, it's a shame because it can help identify people that are in need of a search/arrest/etc and knowing how they look it's a good feature, it has been broken for a couple of weeks i believe
![Screenshot_2](https://user-images.githubusercontent.com/57050500/212802987-7e2f14b3-6cdf-46ae-aaa6-d568400f35cb.png)

**Comment by itsmeow** [comment]
I think you just need to reload the record. It's because the page loaded before the photo finished being sent to the client

**Comment by Fronsis** [comment]
Hmm, how do you reload the record? I've tried opening and closing it a couple of times(when i have to update the crime of someone, etc) and this seems to happen every round, at least on my end.

**Comment by Fronsis** [comment]
> I think you just need to reload the record. It's because the page loaded before the photo finished being sent to the client

Hmm, how do you reload the record? I've tried opening and closing it a couple of times(when i have to update the crime of someone, etc) and this seems to happen every round, at least on my end.

**Comment by Fronsis** [comment]
Still broken.

**Comment by Penwin0** [comment]
The PR which mentioned this issue has nothing to do with it

**Comment by Fronsis** [comment]
Indeed, it was just as a bump to the issue and a hopeful comment, but yeah unfortunately's still present i've yet to manage to get the pictures fixed, and i think newscatter has a very similar issue with photos but i'll do another one for that since it's different

**Comment by Fronsis** [comment]
Well, i'd like to uh.. notify that this is still an issue. @itsmeow i don't know what you mean by reload the records but i've tried everything and recently we were in need of using that to identify a person with X color of hair but we weren't able too and we had to do it the long way (print a wanted person of like 10 different people until we got the person) just to see the pic

Btw, since it's related weren't you working at one point at some sort of tgui upgraded version of this? i remember glimpses of screenshot of that

**Comment by Tsar-Salat** [comment]
Likely fixed by full replacement in #11545
